all/entrypoint.sh

```diff
@@ -136,6 +137,10 @@ while test $# -gt 0; do
             fi
             shift
             ;;
+	--validator-source-relative)
```

Comment:
Is there any reason for the validator files to be source relative while other generated code is not source relative, 
or can the `go-source-relative` flag be used to set the value for the validator?

Reply:
Indeed, I did consider using that flag...however I didn't want to check in something that was not backwards compatible.  
For us I don't believe there would be a reason but I can't speak for everyone.

Reply:
@ido-namely ping

Reply:
Hi @heliuchuan, sorry for the delay.
I'll try to test your changes tonight and if everything works well I'll merge this.